main.go

```diff
@@ -81,8 +81,17 @@ func main() {
 	}
 
 	if *recursiveParameter {
-		for _, dirName := range flag.Args() {
-			recursiveSearch(dirName)
+		if len(flag.Args()) < 1 {
+			currentPath, currentPathErr := os.Getwd()
+			if currentPathErr != nil {
+				fmt.Fprintln(os.Stderr, currentPathErr)
+				os.Exit(2)
+			}
+			recursiveSearch(currentPath)
```

Comment:
Any way not to need `currentPath` here?

Reply:
append current path to flag.Args() if len(flag.Args()) < 1?

Reply:
You're in `/foo/bar`. You'd like to pass `/foo/bar` to a program. How to shorten `/foo/bar`?

Reply:
*/bar?

Reply:
No, because there isn't any `/foo/bar/*/bar`.

Reply:
i dont know how to shorten it then

Reply:
pwd shows me my current path

Reply:
but i used os.Getwd() to get my current path i dont understand how to shorten it even more

Reply:
You're effectively going to read **the current directory**. How to do that as short as possible?

Reply:
as I do at the moment

Reply:
.

Reply:
i dont know how niko also does not know it

Reply:
Tip: .